pom.xml

```
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1857-SNAPSHOT</version>
```

```suggestion
    <version>1.0-SNAPSHOT</version>
```
Reverting to base version after [successful tests in Jenkins](https://jenkins.carlspring.org/blue/organizations/jenkins/strongbox%2Fbuilds%2Fstrongbox/detail/PR-1888/2/pipeline).